engine/sound/src/decoders/decoder_opus.cpp

```diff
@@ -0,0 +1,502 @@
+// Copyright 2020-2024 The Defold Foundation
```

Comment:
I marked a couple of codestyle issues in my review but probably it's worth to run clang-format on that file?